noria-server/mir/src/visualize.rs

```diff
@@ -149,6 +150,24 @@ impl GraphViz for MirNodeType {
                     .join(", ");
                 write!(out, "{} | γ: {}", op_string, group_cols)?;
             }
+            MirNodeType::FilterAggregation {
+                ref on,
+                else_on: _,
+                ref group_by,
+                ref kind,
+                conditions: _,
```

Comment:
You don't need all of these `: _`. Just list the fields you want, and then add `, ..`.

Reply:
Done.